debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresChangeRecordEmitter.java

```diff
@@ -85,7 +85,12 @@ protected Operation getOperation() {
     @Override
     public void emitChangeRecords(DataCollectionSchema schema, Receiver receiver) throws InterruptedException {
         schema = synchronizeTableSchema(schema);
-        super.emitChangeRecords(schema, receiver);
+        Operation operation = getOperation();
+        Set<Operation> skippedOperations = connectorConfig.getSkippedOps();
```

Comment:
I think this can be done in the constructor.

Reply:
Did you mean that should I set the `skippedOperations` on the constructor?

Reply:
Exactly. I think it doesn't have to be fetched for every `emitChangeRecords(...)` call, right?

Reply:
Yeah. you're right